django_celery_results/urls.py

```diff
@@ -24,4 +24,14 @@
         views.task_status,
         name='celery-task_status'
     ),
+    url(
```

Comment:
better use path()

Reply:
@auvipy would you prefer that I just use path for the GroupResult views, or switch to `path` for the AsyncTask views as well?

Reply:
all

Reply:
I've converted all the url's to path. One side effect of this is that I had to create a custom path converter, since the regex for task/group id's is a little different then the built in converters